### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by Evsio0n**
Add a solution where use wsl remotely by using OpenSSH on windows

### CODE REVIEW

**Comment by Evsio0n** [comment]
@hyoshioka0128  @partychen  pinging for review

**Comment by hyoshioka0128** [comment]
@Evsio0n - LGTM. 
I'm not a Microsoft employee, so any help would be appreciated. Thanks.

**Comment by mattwojo** [comment]
Looks good to me. @craigloewen-msft -- ok to merge this?

File: WSL/troubleshooting.md [hunk 1]
```diff
@@ -445,6 +445,20 @@ options = metadata,uid=1000,gid=1000,umask=0022
...
+
+### Fails to use wsl remotely by using OpenSSH on windows
+
+If you are using openssh-server on windows and tring to access wsl remotely, you many see this error:
+
+```cmd
+The file cannot be accessed by the system.
+```
+
+It's a [known issue](./store-release-notes.md#known-issues), and solution to this is to change wsl version to wsl1.
```
Comment (craigloewen-msft) [inline WSL/troubleshooting.md:458]:
```suggestion
It's a [known issue](./store-release-notes.md#known-issues), 
when using the Store version of WSL. You can work around this today by using WSL 1, or by using the in-Windows version of WSL. See https://aka.ms/wslstoreinfo for more info.
```

**Pull request merged**